server/utils/podcastUtils.js

```diff
@@ -85,7 +85,7 @@ function extractEpisodeData(item) {
     episode.descriptionPlain = stripHtml(episode.description || '').result
   }
 
-  var arrayFields = ['title', 'pubDate', 'itunes:episodeType', 'itunes:episode', 'itunes:author', 'itunes:duration', 'itunes:explicit', 'itunes:subtitle']
+  var arrayFields = ['title', 'pubDate', 'itunes:episodeType', 'season', 'itunes:episode', 'itunes:author', 'itunes:duration', 'itunes:explicit', 'itunes:subtitle']
```

Comment:
I pulled the feed for command line heros and see that the tag is actually `itunes:season` and not `season`. I'm not sure if you tested this already and it scraped properly but I don't see the tag `season`.
![image](https://user-images.githubusercontent.com/67830747/166836262-3677db0c-0373-4ca5-868f-595a350dd045.png)

Reply:
fyi the itunes part of the string gets stripped out when returning from that function.

Reply:
Good catch! I'll get this fixed

Reply:
It's verking!